recipes/pwtools/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "pwtools" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/elcorto/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 54ad5275372c6c41896020b2600cee785c56180d063b76b2c26aab588ac566d3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win or py2k]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - matplotlib
```

Can you  `matplotlib-base` here to avoid the Qt dependency?
```suggestion
    - matplotlib-base
```